rocketman/templates/streams/cards_block.html

```diff
@@ -0,0 +1,31 @@
+{% load wagtailimages_tags %}
+<!--card block -->
+<div class="row justify-content-center">
+  {% for card in value.cards %}
+    <div class="col-md- d-flex aligin-items-stretch{% if not forloop.first %} mt-4 mt-md-0{% endif %}">
+<div class="card mb-3 text-center">
+  <a href="{% if card.link.internal_page %}{{ card.link.internal_page.url}}{% elif card.link.external_link %}{{ card.link.external_link }}{% else%}#{% endif %}">
+    <picture>
+      {% image card.image fill-570x370 as img %}
+      <img src="{{ img.url }}" class="card-img-top" alt="{{ img.alt }}">
+    </picture>
+  </a>
+  <div class="card-body py-3 px-3">
+    <a href="{% if card.link.internal_page %}{{ card.link.internal_page.url}}{% elif card.link.external_link %}{{ card.link.external_link }}{% else%}#{% endif %}">
+      <h5 class="card-title">{{ card.title }}</h5>
+      {% if card.text %}
+      <p class="card-text">{{ card.text }}</p>
+      {% endif %}
+    </a>
+
+    <p class="card-text">
+      <a href="{% if card.link.internal_page %}{{ card.link.internal_page.url}}{% elif card.link.external_link %}{{ card.link.external_link }}{% else%}#{% endif %}" class="btn btn-link d-inline">
+        {{card.link.link_text}}
+      </a>
+    </p>
+
+      </div>
+    </div>
+  </div>
```

Comment:
This HTML indenting could use some work. Its hard to quickly review it to make sure all tags are closing.